docs/CODEOWNERS

```diff
@@ -0,0 +1,12 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+*       @ottodevs @Quazia @rkzel @topocount @chadoh @Schwartz10
```

Comment:
In addition to this global list of code owners, we could also add frontend code owners. Would that be useful? @ottodevs I know you would want to be on that list. Do we want that? And who all should be on such a list?

Reply:
Generally I don't think it's good to rely on @Schwartz10 for reviews on TPS